Applications/Utils/MeshEdit/CMakeLists.txt

```diff
@@ -26,6 +26,20 @@ if(QT4_FOUND)
 
 	set_target_properties(moveMeshNodes PROPERTIES FOLDER Utilities)
 
+	add_executable(MapGeometryToMeshSurface
+		MapGeometryToMeshSurface.cpp )
+	target_link_libraries(MapGeometryToMeshSurface
+		MeshLib
+		MeshGeoToolsLib
+		FileIO
+		InSituLib
+		${CATALYST_LIBRARIES}
+		${QT_LIBRARIES}
```

Comment:
The catalyst libraries are necessary for the MeshLib. Only the part of Qt used for vtk is necessary. So I removed `${QT_LIBRARIES}`.